addon/ng2/blueprints/component/index.js

```diff
@@ -1,29 +1,26 @@
-var stringUtils = require('ember-cli/lib/utilities/string');
 var dynamicPathParser = require('../../utilities/dynamic-path-parser');
 
 module.exports = {
   description: '',
 
-  normalizeEntityName: function(entityName) {
+  normalizeEntityName: function (entityName) {
     var parsedPath = dynamicPathParser(this.project, entityName);
-    
+
     this.dynamicPath = parsedPath;
     return parsedPath.name;
   },
 
-  locals: function(options) {
-    return {
-      dynamicPath: this.dynamicPath.dir
-    };
+  locals: function () {
+    return {dynamicPath: this.dynamicPath.dir};
```

Comment:
Place spaces after `{` and before `}` please. I prefered the original one though :/